src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ CHANGELOG
  * Add support for `ConfigBuilder` in the `PhpFileLoader`
  * Add `ContainerConfigurator::env()` to get the current environment
  * Add `#[Target]` to tell how a dependency is used and hint named autowiring aliases
+ * Add support of PHP 8.1 enumerations
```

Comment:
We're going to merge this in 4.4 as a bugfix, please rebase and remove this line.